test/CMakeLists.txt

```
@@ -0,0 +1,60 @@
+if(NOT TARGET gtest OR NOT TARGET gmock)
+  # Prevent overriding the parent project's compiler/linker settings on Windows
+  set(gtest_force_shared_crt # cmake-lint: disable=C0103
+      ON
+      CACHE BOOL "" FORCE)
+  add_subdirectory(
+    "${PROJECT_SOURCE_DIR}/extern/qcec/extern/qfr/extern/dd_package/extern/googletest"
+    "extern/qcec/extern/qfr/extern/dd_package/extern/googletest" EXCLUDE_FROM_ALL)
+  add_subdirectory(
+          "${PROJECT_SOURCE_DIR}/extern/ddsim/extern/qfr/extern/dd_package/extern/googletest"
+          "extern/ddsim/extern/qfr/extern/dd_package/extern/googletest" EXCLUDE_FROM_ALL)
```

this just needs to be added once. This should actually result in an error since you are trying to include googletest twice.
```suggestion
  add_subdirectory(
    "${PROJECT_SOURCE_DIR}/extern/qcec/extern/qfr/extern/dd_package/extern/googletest"
    "extern/qcec/extern/qfr/extern/dd_package/extern/googletest" EXCLUDE_FROM_ALL)
```